.github/workflows/issue-branch.yml

```
@@ -1,85 +1,113 @@
-    types: [opened]
+  pull_request:
+    types: [opened, ready_for_review]
 
 jobs:
-  create-branch:
+  build-design-system:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)